src/assets/scripts/client/base/BaseModel.js

```diff
@@ -45,4 +49,22 @@ export default class BaseModel {
     reset() {
         throw new TypeError('BaseModel#reset method must be implemented by the class extending BaseModel');
     }
+
+    /**
+     * This will verify if the given argument is a string otherwise it will return 'Base Model'
+     *
+     * @for BaseModel
+     * @method veriftyModelName
+     * @param {string / Object}
+     */
+    buildIdPrefix(modelName) {
```

Comment:
By private do you mean just giving it a naming scheme with an _ in front of the method name and adding @private in the doc blocks?